lib/text/translationutils.flow

```diff
@@ -1,11 +1,9 @@
-import string;
 import text/languages;
-import text/translation_fixup;
-import text/translationapi; // ContentUnit
 import fs/filesystem; // fileNameOnly
 import formats/json/json;
-import math/stringmath;
+import text/translationapi; // ContentUnit
 import ui/fontmapping; // getLang
+import formats/xml;
```

Comment:
We minimized imports and excluded xml stuff for simple strings. It is why there was simplified xmlEscape function.
Lightweight application itself don't  need xml for translation - it only loads json with simple strings.
To keep this approach I think we need to restore this here and exclude xml import from this utils file.

Reply:
This small amount is OK to have in flow9.

Reply:
Import xml - OK if no problem with app file size now. But my question was not about this import but about functions like this:

xliffNode2WigiLightString(node : XmlNode, level : int, maybeParent : Maybe<XmlNode>) -> string;

This is utilities for wigi format but flow9 as open source knows nothing about wigi so it seems to me it should be placed in innovation repo like translationwikisupport.flow.